chainerx/_docs/routines.py

```diff
@@ -1342,6 +1342,27 @@ def _docs_math():
     output array to the input array ``x``.
 
 .. seealso:: :data:`numpy.arctan`
+""")
+
+    _docs.set_doc(
+        chainerx.arctan2,
+        """arctan2(x1, x2)
+Element-wise arc tangent of x1/x2 choosing the quadrant correctly.
+
+Args:
+    x1 (~chainerx.ndarray): Input array.
+    x2 (~chainerx.ndarray): Input array.
+
+Returns:
+    :class:`~chainerx.ndarray`: Returns an array where each element
+    represents theta in the range [-pi, pi], such that x1 = r sin(theta),
+    x2 = r cos(theta) for some r > 0.
```

Comment:
```
Returns:
    :class:`~chainerx.ndarray`: Returns an array where each element
    represents :math:`\\theta` in the range :math:`[-\\pi, \\pi]`, such
    that :math:`x_1 = r \\sin(\\theta)` and :math:`x_2 = r \\cos(\\theta)`
    for some :math:`r > 0`.
```

Reply:
Oh thank you.Sorry, I don't know much about Sphinx notations. Thank You.